packages/manager/modules/pci/src/projects/project/edit/edit.html

```diff
@@ -22,6 +22,6 @@
     <div class="mt-4 pt-4 border-top">
         <h2 data-translate="pci_projects_project_edit_remove"></h2>
         <p data-translate="pci_projects_project_edit_remove_description"></p>
-        <oui-button variant="secondary" text="{{ ::'pci_projects_project_edit_remove' | translate }}" disabled="$ctrl.loading.submit"></oui-button>
+        <a class="oui-button oui-button_secondary" data-ui-sref="pci.projects.project.edit.remove" variant="secondary" data-translate="pci_projects_project_edit_remove"></a>
```

Comment:
Not sure that the variant attribut is still needed.

```suggestion
        <a class="oui-button oui-button_secondary" data-ui-sref="pci.projects.project.edit.remove" data-translate="pci_projects_project_edit_remove"></a>
```